lib/image_science.rb

```diff
@@ -13,6 +13,89 @@
 class ImageScience
   VERSION = '1.3.0'
 
+  FREE_IMAGE_FORMAT = {
+    FIF_UNKNOWN: -1,
+    FIF_BMP: 0,
+    FIF_ICO: 1,
+    FIF_JPEG: 2,
+    FIF_JNG: 3,
+    FIF_KOALA: 4,
+    FIF_LBM: 5,
+    FIF_IFF: 5,
+    FIF_MNG: 6,
+    FIF_PBM: 7,
+    FIF_PBMRAW: 8,
+    FIF_PCD: 9,
+    FIF_PCX: 10,
+    FIF_PGM: 11,
+    FIF_PGMRAW: 12,
+    FIF_PNG: 13,
+    FIF_PPM: 14,
+    FIF_PPMRAW: 15,
+    FIF_RAS: 16,
+    FIF_TARGA: 17,
+    FIF_TIFF: 18,
+    FIF_WBMP: 19,
+    FIF_PSD: 20,
+    FIF_CUT: 21,
+    FIF_XBM: 22,
+    FIF_XPM: 23,
+    FIF_DDS: 24,
+    FIF_GIF: 25,
+    FIF_HDR: 26,
+    FIF_FAXG3: 27,
+    FIF_SGI: 28,
+    FIF_EXR: 29,
+    FIF_J2K: 30,
+    FIF_JP2: 31,
+    FIF_PFM: 32,
+    FIF_PICT: 33,
+    FIF_RAW: 34,
+    FIF_WEBP: 35,
+    FIF_JXR: 36
+  }
+
+  CONTENT_TYPES = {
+    FIF_UNKNOWN: 'image',
+    FIF_BMP: 'image/bmp',
+    FIF_ICO: 'image/x-icon',
+    FIF_JPEG: 'image/jpeg',
+    FIF_JNG: 'image',
+    FIF_KOALA: 'image',
+    FIF_LBM: 'image',
+    FIF_IFF: 'image',
+    FIF_MNG: 'image',
+    FIF_PBM: 'image/x-portable-bitmap',
+    FIF_PBMRAW: 'image/x-portable-bitmap',
+    FIF_PCD: 'image',
+    FIF_PCX: 'image/x-pcx',
+    FIF_PGM: 'image/x-portable-greymap',
+    FIF_PGMRAW: 'image/x-portable-greymap',
+    FIF_PNG: 'image/png',
+    FIF_PPM: 'image/x-portable-pixmap',
+    FIF_PPMRAW: 'image/x-portable-pixmap',
+    FIF_RAS: 'image/cmu-raster',
+    FIF_TARGA: 'image/x-targa',
+    FIF_TIFF: 'image/tiff',
+    FIF_WBMP: 'image/vnd.wap.wbmp',
+    FIF_PSD: 'application/octet-stream',
+    FIF_CUT: 'image/x-cut',
+    FIF_XBM: 'image/xbm',
+    FIF_XPM: 'image/xpm',
+    FIF_DDS: 'image/vnd-ms.dds',
+    FIF_GIF: 'image/gif',
+    FIF_HDR: 'image/vnd.radiance',
+    FIF_FAXG3: 'image/fax-g3',
+    FIF_SGI: 'image',
+    FIF_EXR: 'image/x-exr',
+    FIF_J2K: 'image/jp2',
+    FIF_JP2: 'image/jp2',
+    FIF_PFM: 'application/octet-stream',
+    FIF_PICT: 'image/x-pict',
+    FIF_RAW: 'image/raw',
+    FIF_WEBP: 'image/webp',
+    FIF_JXR: 'image/jxr'
+  }
```

Comment:
Maybe you can come up with a way to infer this from FreeImage. I'm not a C developer so it would be out of my depth. I did find a couple of methods though in FreeImage that could probably be wrapped, such as `FreeImage_GetFIFMimeType` and `FreeImage_GetFIFFromMime`